### ISSUE TITLE
Fail to respond to ping in Azure App Service deployment

### ISSUE THREAD

**Issue by JamesPHoughton** [empiricaly/empirica#271]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I'm trying to deploy on Azure App Service, which expects the server to respond to a ping on port 3000 after startup. It looks to me that the server is starting (ie, container logs show "callbacks started" message), but not responding to the ping:

```
Container ... didn't respond to HTTP pings on port: 3000, failing site start. See container logs for debugging.
```

I'm not sure if this is because I don't have the port mapping configured correctly or if empirica isn't set up to respond to the ping.

### Steps To Reproduce

Here's the built container I'm trying to deploy: https://github.com/Watts-Lab/deliberation-empirica/pkgs/container/deliberation-empirica 

### Empirica Version

```text
SHA:     47c52f3
Build:   249
Branch:  bugfix-merge-build-release
Time:    2023-02-08T09:59:15Z

Client:  1.0.6 ->  -> 
Server:  1.0.6 -> 1.0.0-rc.23 -> 1.0.0-rc.23
```

### What OS are you seeing the problem on?

Linux

### Relevant log output

```shell
2023-04-01T14:03:03.240189150Z Starting empirica
2023-04-01T14:03:03.432645119Z {"level":"info","target":"/tmp/c5472068acadc3e4","time":"2023-04-01T14:03:03.432446817Z","message":"unbundle: installing"}
2023-04-01T14:03:03.718382140Z {"level":"info","time":"2023-04-01T14:03:03.718253238Z","message":"serve: server started"}
2023-04-01T14:03:03.817164353Z {"level":"info","time":"2023-04-01T14:03:03.817004452Z","message":"empirica: server started"}
2023-04-01T14:03:12.242001311Z [90m14:03:12.238[0m [32mINF[0m callbacks: started
/home/LogFiles/2023_04_01_lw0sdlwk00025T_docker.log  (https://deliberation.scm.azurewebsites.net/api/vfs/LogFiles/2023_04_01_lw0sdlwk00025T_docker.log)
2023-04-01T14:05:06.805Z INFO  - Waiting for response to warmup request for container deliberation_0_90300144. Elapsed time = 122.7982093 sec
2023-04-01T14:05:22.050Z INFO  - Waiting for response to warmup request for container deliberation_0_90300144. Elapsed time = 138.0427789 sec
2023-04-01T14:05:37.610Z INFO  - Waiting for response to warmup request for container deliberation_0_90300144. Elapsed time = 153.6027395 sec
2023-04-01T14:05:55.066Z INFO  - Waiting for response to warmup request for container deliberation_0_90300144. Elapsed time = 171.0586357 sec
2023-04-01T14:06:10.663Z INFO  - Waiting for response to warmup request for container deliberation_0_90300144. Elapsed time = 186.6562441 sec
2023-04-01T14:06:27.553Z INFO  - Waiting for response to warmup request for container deliberation_0_90300144. Elapsed time = 203.5460437 sec
2023-04-01T14:06:43.724Z INFO  - Waiting for response to warmup request for container deliberation_0_90300144. Elapsed time = 219.7171917 sec
2023-04-01T14:06:54.901Z ERROR - Container deliberation_0_90300144 for site deliberation did not start within expected time limit. Elapsed time = 230.8940064 sec
2023-04-01T14:06:54.918Z ERROR - Container deliberation_0_90300144 didn't respond to HTTP pings on port: 3000, failing site start. See container logs for debugging.
2023-04-01T14:06:54.968Z INFO  - Stopping site deliberation because it failed during startup.
```

### Anything else?
Some debugging I've tried:
https://learn.microsoft.com/en-us/answers/questions/168746/container-didnt-respond-to-http-pings-on-port-8080

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by npaton** [comment]
I don't really know tbh. This app environment is foreign to me. Did you add the EXPOSE in your Dockerfile?

**Comment by JamesPHoughton** [comment]
Being quite ignorant, I wasn't sure if I the problem was with the Azure setup, or if we needed to do something special to make Empirica respond to a ping. (Is this a standard thing that web server code does?) Turns out I was just setting it up wrong, and Empirica does respond to pings if you send the ping to the right place.

In the end, I had to add the expose in the dockerfile and also [tell Azure to send requests explicitly to port 3000](https://learn.microsoft.com/en-us/azure/app-service/configure-custom-container?tabs=debian&pivots=container-linux#configure-port-number), which makes sense, it just wasn't obvious. 

Not sure I like Azure at the moment...
